kale/loaddata/sampler.py

```diff
@@ -334,6 +357,7 @@ class DomainBalancedBatchSampler(BalancedBatchSampler):
 
     def __init__(self, dataset, batch_size):
         # call to __init__ of super class will generate class balanced sampler, do not do it here
+        super().__init__(dataset, batch_size)
```

Comment:
This line conflicts with the comment above. Please check and update the comment or the code accordingly.